zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -966,7 +966,11 @@ angular.module('zeppelinWebApp')
       }
       return '';
     }
-    var desc = 'Took ' + (timeMs/1000) + ' seconds';
+    var user = 'anonymous';
+    if (pdata.authenticationInfo != null && pdata.authenticationInfo.user != null) {
+      user = pdata.authenticationInfo.user;
+    }
+    var desc = 'Took ' + (timeMs/1000) + ' seconds. Last updated by ' + user + ' at ' + pdata.dateUpdated;
```

Comment:
I think that in the case of Zeppelin not using the auth, we should use the previous message (without anonymous)